app/views/layouts/application.html.haml

```diff
@@ -27,7 +27,6 @@
     = yield :header_tags
 
   - body_classes ||= @body_classes || ''
-  - body_classes += ' reduce-motion' if current_account&.user&.setting_reduce_motion
```

Comment:
We're not currently using this in the CSS, but we can add it back in later if need be